src/BaselineOfIDE.package/BaselineOfIDE.class/instance/baseline..st

```diff
@@ -66,8 +66,6 @@ baseline: spec
 		spec package: 'Zodiac-Extra'.	
 		spec package: 'Zinc-Zodiac'.	
 			
-		spec package: 'Metacello-GitBasedRepository'.	
```

Comment:
This is wrong. Somehow the merge is removing this line and this line should stay.